src/gennylib/include/gennylib/conventions.hpp

```diff
@@ -423,6 +441,56 @@ struct convert<genny::RateSpec> {
     }
 };
 
+/**
+ * Convert between YAML and genny::PercentileRateSpec
+ *
+ * The YAML syntax accepts [genny::Integer]%
+ * The syntax is interpreted as a percentage of the max throughput.
+ */
+template <>
+struct convert<genny::PercentileRateSpec> {
```

Comment:
Let's also update one of the `docs` workloads to use this new percentage syntax and explain how it works.

Reply:
Done, added a comment in `HelloWorld.yml` explaining.